src/sentry/static/sentry/app/views/releases/list/organizationReleases/index.jsx

```diff
@@ -100,34 +86,42 @@ class OrganizationReleases extends React.Component {
     );
   }
 
-  render() {
-    if (!new Set(this.props.organization.features).has('sentry10')) {
-      return this.renderNoAccess();
-    }
+  renderError() {
+    return this.renderBody();
+  }
+
+  renderLoading() {
+    return this.renderBody();
```

Comment:
imo using AsyncView in situations like this doesn't add much benefit

Reply:
I think there's benefit in having a consistent way to handle a view that requires an API call when it's mounted. I agree it's not very straightforward in this situation when we're trying to "customize" loading (`AsyncView` that takes a render prop would fit better in this situation) and we may need to revisit this pattern if we find ourselves not using the default loading pattern more frequently.